examples/rust_with_cffi/noxfile.py

```diff
@@ -7,6 +7,6 @@
 
 @nox.session()
 def test(session: nox.Session):
-    session.install(SETUPTOOLS_RUST, "pytest", "cffi")
+    session.install(SETUPTOOLS_RUST, "pytest", "cffi", "distutils-extra")
```

Comment:
I don't think python-distutils-extra is on pypi (or at least I couldn't find it). I don't really know nox but if it just wraps pip, I suppose you could use the tarball: `https://launchpad.net/python-distutils-extra/trunk/2.39/+download/python-distutils-extra-2.39.tar.gz`

Reply:
Ehh, I can't manage to verify the crash with or without that tarball installed locally.

As you've verified the patch works, I'm happy 👍